app/src/main/java/com/example/albumsapp/parser/PhotoColorParser.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.albumsapp.parser
+
+import android.graphics.Color
+import java.util.*
+
+class PhotoColorParser(private val url: String) {
+    fun parse(): Int {
+        var color = "#" + url.takeLastWhile { it != '/' }.uppercase(Locale.ROOT)
+        while (color.length < 7) {
+            color += "0"
```

Comment:
hmm, why do we add `0` to the end of `color`? Why not prepend it on the beginning of the color string?

Reply:
i add 0 to the end of color, because some urls have length smaller than needed and we can't parse it

Reply:
i understand it, but why to the end? Usually we add 0 before numbers and it does not change the value.
For example `00000000000018` == `18`, but `1800000000` != `18`